src/Widgets/Alarm.vala

```diff
@@ -81,11 +81,21 @@ public class Hourglass.Widgets.Alarm : Gtk.ListBoxRow {
 
     private string make_date_label () {
         var comp = new GLib.DateTime.now_local ();
-        if (Granite.DateTime.is_same_day (time, comp)) {
+        if (Granite.DateTime.is_same_day (time, comp) && repeat.length == 0) {
+            return _("Today");
+        }
+
+        int today = comp.get_day_of_week () != 7 ? comp.get_day_of_week () : 0;
+        if (today in repeat) {
```

Comment:
```suggestion
        int today = comp.get_day_of_week () != 7 ? comp.get_day_of_week () : 0;
        if (
            Granite.DateTime.is_same_day (time, comp) && repeat.length == 0 ||
            today in repeat
        ) {
```

This might be looks a little bit complicated but can DRY the string "Today"

Reply:
Good eye! Thanks for the changes.